docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
-[//]: # (title: Output formats supported by Kotlin Notebooks)
+[//]: # (title: Output formats supported by Kotlin Notebook)
 
-There will be examples for each output.
+Kotlin Notebook supports a variety of output types, including text, HTML, and images. With the help of external libraries,
+you can expand your output options and visualize your data with charts, spreadsheets, and more.
 
-One key feature that makes notebooks stand out is the variety of output types they support:
+## Texts
 
-## Text
+### Plain text
 
-This is the simplest type of output, and it includes printed statements, variable values, or any text-based output from your code. If a cell’s result doesn’t fall under one of the categories below, it will be printed as text via the toString() method.
+The simplest output type is plain text. It's used in printed statements, variable values, or any text-based output from
+your code:
 
-## Rich text
+```kotlin
+val a1: Int = 1
+val a2: Int = 2
+var a3: Int? = a1 + a2
 
-Markdown cells produce rich text HTML output, offering support for lists, font styles, code blocks, and more.
+"My answer is $a3"
+```
 
-## HTML markup
+![Plain text code output](plain-text-output.png){width=300}
 
-Jupyter notebooks can render HTML directly, enabling rich text formatting, the use of tables, or even the embedding of websites.
+* If a cell's result doesn't match with other output types, it will be printed as plain text using the `toString()` function.
+* If your code contains errors, Kotlin Notebook displays an error message and a traceback, providing insights for debugging.
+
+### Rich text
+
+Use Markdown types of cells to get rich text output. It supports various types of formatting with Markdown and HTML markup.
```

Comment:
I would say "Cells of Markdown type". By the way, markdown cells are NOT outputs, I'm not sure it's clear now